document release notes by pitching changie as a tool (#32)
scality/citadel: docs/release/disconnecting.md

* Create a build chain that outputs the desired installation code and packages.
* Trigger integration tests that ensure all selected components interoperate effectively.
* Document the project's services, and how to build, test, configure, run, monitor, and contribute to the project.

> * Offer [release notes](./release-notes.md) that document changes in the product's function and features.

Reviewer: Correcting my own mistakes :sweat_smile: 

```suggestion
* Offer [release notes](./release-notes.md) that document changes in the project's function and features.
```

* Create a build chain that outputs the desired installation code and packages.
* Trigger integration tests that ensure all selected projects interoperate effectively.
* Document the product's services; how to install, configure, run, and monitor the product; and how to solve product issues.

> * Offer [release notes](./release-notes.md) that document changes in the project's function and features.

Reviewer: As much as I would love to see this change on the product level, Citadel may not be the place to request it as it's mainly for the object squad. At least for now :sweat_smile: 

So maybe I made a mistake of adding this release note line on the product repository.

> * Offer [release notes](./release-notes.md) that document changes in the project's function and features.

Reviewer: also this mistake is on me
```suggestion
* Offer [release notes](./release-notes.md) that document changes in the products's function and features.
```

Author: ```suggestion
* Offer release notes that document changes in the product's function and features.
```
